mne/gui/tests/test_stc.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
```

Comment:
this GUI is not for all STCs. I would not call this file test_stc.py but rather test_vol_tfr_stc.py. This applies also the the _stc.py file that should be renamed.